NEWS.md

```diff
@@ -3,6 +3,7 @@ plumber (version)
 
 ### New features
 
+* Add support for @plumber tag that modify plumber router programmatically using expression of the form function(pr) {pr$...}.
```

Comment:
I'd like to add `@blairj09` as he helped me walk through the many design phases of this implementation.

```suggestion
* Add support for `#' @plumber` tag that modify plumber router programmatically using expression of the form `function(pr) {....}`. (@meztez and @blairj09, #568)
```